Automattic/newspack-newsletters
# Fix colors handling

### All Submissions:






### Changes proposed in this Pull Request:



Closes #93 

### How to test the changes in this Pull Request:

1. Create a newsletter with columns and groups. Set colors on them (background colors, text colors, both custom and from the presets)
2. Save post and view the preview
3. Observe that colors are rendered correctly

### Other information:
